front/src/modules/ui/table/states/tableViewsState.ts

```diff
@@ -0,0 +1,50 @@
+import { atom, atomFamily, selectorFamily } from 'recoil';
+
+export type TableView = { id: string; name: string };
+
+export const tableViewsState = atomFamily<TableView[], string>({
```

Comment:
currently, we put one state per file, and put types in types folder.
I think these tableViews should be part of a views module and not tableViews